osrelease/scanner.go

```diff
@@ -123,21 +139,48 @@ func parse(ctx context.Context, r io.Reader) (*claircore.Distribution, error) {
 
 		switch key {
 		case "ID":
+			log.Debug().Msg("found ID")
```

Comment:
I think these debug messages should go in favor or a single one that summarizes the fully realized dist struct.